examples/inverter_example.py

```diff
@@ -1,15 +1,17 @@
-""" Example code to run the forecast"""
 import pandas as pd
-from datetime import datetime
+from datetime import datetime, timezone
 from quartz_solar_forecast.forecast import run_forecast
 from quartz_solar_forecast.pydantic_models import PVSite
-from datetime import datetime, timezone
+import os
+import typer
```

Comment:
Does typer need to be added to the requirements.txt file as well?

Reply:
> Does typer need to be added to the requirements.txt file as well?

Yup, updated